packages/react/src/components/index.ts

```diff
@@ -36,6 +36,15 @@ export * from './hrefprops';
 // Ionic Animations
 export { CreateAnimation } from './CreateAnimation';
 
+// Hooks
+export * from '../hooks/useIonActionSheet';
```

Comment:
Can you avoid using * here and just export what we know should be exported? Ultimately it's the same thing, but with more code, but like to play it safe we don't accidentally export anything we shouldn't be.